### ISSUE TITLE
Chart version does not match the Docker image tag

### ISSUE THREAD

**Issue by izeau** [woodpecker-ci/woodpecker#863]
### Component

server, agent

### Describe the bug

In the `woodpecker/woodpecker-server` & `woodpecker/woodpecker-agent` Helm charts, the Docker image tag defaults to the `.Chart.AppVersion` value – which is currently `0.15.0`. The Docker images however are published with a `v`-prefixed tag, such as `v0.15.0`. As a result, Kubernetes is unable to pull the corresponding images unless `image.tag` is forced to `v0.15.0` (mind the `v`) when installing the charts.

I’ll gladly submit a PR for this issue if it’s OK with you.

### System Info

```shell
{"source":"https://github.com/woodpecker-ci/woodpecker","version":"0.15.0"}
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy).

**Comment by anbraten** [comment]
Ohh damn. I think the docker tag scheme was changed at some point and we missed adjusting it in the helm chart. TBH I would like to move to the version without the v in general as the v just makes things more complicated and has no benefit I am aware of, but maybe that's to much migration work 🤔😭

**Comment by anbraten** [comment]
I created a fix for it #882 and already updated it for the latest release `0.15.1` to be working fine.

**Comment by izeau** [comment]
Works perfectly, thanks :)
